src/js/components/search-bar.js

```diff
@@ -33,19 +37,22 @@ export class SearchBar extends HTMLElement {
   }
 
   search(value) {
-    location.href = `/search/?search=${value}`;
+    location.href = `/search/?search=${value}&action=${this.actionQuery}`;
   }
 
   render() {
     const searchbar = htmlUtilities.createHTML("div", "relative text-dark-200");
     if (this.type === "sidebar-search") {
       searchbar.classList.add("mb-4");
+    } else {
+      searchbar.classList.add(..."mx-6 xs:mx-0".split(" "));
```

Comment:
This is a little more complicated than it needs to be

```suggestion
      searchbar.className += " mx-6 xs:mx-0"
```